label_sync/labels.yaml

```diff
@@ -627,3 +626,15 @@ labels:
     target: both
     prowPlugin: milestonestatus
     addedBy: "members of a configurable github team. default: [@kubernetes/kubernetes-milestone-maintainers](https://github.com/orgs/kubernetes/teams/kubernetes-milestone-maintainers/members)"
+repos:
+  kubernetes/kubernetes:
```

Comment:
ops nit: AFAICT if I were to run this tool with `--orgs kubernetes-incubator` then this repo would never be touched, which might be confusing; we need to find a way to surface this, preferably at PR time

Reply:
indeed, so how would you like to use that option, if it existed?

Reply:
I guess the "correct" thing to do would be to rewrite the loops such that label_sync is iterating on repos instead of iterating on orgs.  Another option would be to do two passes, one org iteration, and then one repo iteration for any specific repos called out, skipping any that org iteration caught.

At the very least it seems Configuration.validate() should error out or warn if a repo is present in the config that's not going to get picked up due the `--orgs` flag.

The scenario I want to avoid or improve troubleshooting visibility for is: 
- a PR is merged adding labels for "kubernetes-sigs/foo"
- the labels don't get added
- after much head scratching someone (who has kubectl access to our prow cluster) finally notices our cluster is running a label_sync job with `--orgs kubernetes`

Reply:
It seems the whole utility begs for a much deeper rewrite, as for example I see nowhere that the default org is "kubernetes".
While at it, do you want to support as well org scoped labels?
Just to validate the way you call it:
- with nothing: defaults to --orgs "kubernetes"
- with --orgs "kubernetes-incubator" for a specific org
- with --orgs "kubernetes,kubernetes-incubator" for 2 orgs
- with --repos "kubernetes/kubernetes"

Do you like it?

Reply:
The org flag defaults to nothing, you have to explicitly set it or nothing happens.  [We currently run it with `--orgs=kubernetes,kubernetes-sigs`](https://github.com/kubernetes/test-infra/blob/master/label_sync/cluster/label_sync_cron_job.yaml#L35).  

Org-scoped labels sound cool but let's not blow scope here.

It's the silent failure that I'm concerned about.  If someone wanted to use this utility on their own org/repo it's maybe not clear or intuitive that they would have to run with `--orgs` set to every org prefix of every repo in their config for it to take effect for those repos.  This is absolutely something validate() can warn about at runtime, and it's already iterating through all the repos anyway.